list.js

```diff
@@ -0,0 +1,185 @@
+
+var hobbie_names = ["Dancing", "Listening Music", "Cricket", "Singing", "Fooseball", "Reading Novels"];
+var goal_names = ["Gymming", "Tech Group", "UPSC", "Branch Change", "Internship"];
+
+function n_2_i_h(a){
+  var res = hobbie_names.indexOf(a);
+  return res;
+}
+function n_2_i_g(a){
+  var res = goal_names.indexOf(a);
```

Comment:
use better function names

Reply:
Function name changed to name_to_index_hobbie